lib/sdn_epc/converter.ex

```diff
@@ -0,0 +1,107 @@
+defmodule SdnEpc.Converter do
+  require SdnEpc.OfpmRecord
+
+  @ofp_version Application.get_env(:sdn_epc, :ofp_version)
+
+  def convert({:packet_in, xid, body}) do
```

Comment:
I can't see your idea. I would do it in that way: export the piece of code you mentioned to separate function like build_ofp_msg/2 then call this function from convert/1 passes xid and a particular body (record) as its parameters. Is this correct?